cost/aws/savings_plan/recommendations/aws_savings_plan_recommendations.pt

```diff
@@ -208,19 +208,57 @@ policy "aws_sp_recommendations" do
 ### Savings Plan Purchase Option: {{parameters.param_payment_option}}
 ### Lookback Period: {{parameters.param_days}}
 
-| Estimated Monthly Savings | Estimated Savings Percentage | Estimated Savings Plan Cost | Hourly Commitment to Purchase | Upfront Cost | Instance Family | Region | Current Avg Hourly OnDemand Spend | Current Max Hourly OnDemand Spend | Current Min Hourly OnDemand Spend | Offering ID |
-| ------------------------- | ---------------------------- | --------------------------- | ----------------------------- | ------------ | --------------- | ------ | --------------------------------- | --------------------------------- | --------------------------------- | ----------- |
-{{ range data -}}
-| {{.estimatedMonthlySavingsAmount}} ({{.currencyCode}}) | {{.estimatedSavingsPercentage}}% | {{.estimatedSavingsPlanCost}} ({{.currencyCode}}) | {{.hourlyCommitmentToPurchase}} ({{.currencyCode}}) | {{.upfrontCost}} ({{.currencyCode}}) | {{.instanceFamily}} | {{.region}} | {{.currentAverageHourlyOnDemandSpend}} ({{.currencyCode}}) | {{.currentMaximumHourlyOnDemandSpend}} ({{.currencyCode}}) | {{.currentMinimumHourlyOnDemandSpend}} ({{.currencyCode}}) | {{.offeringId}}
-{{ end -}}
-
 ###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
 EOS
+    export do
+      field "estimatedMonthlySavingsAmount" do
+        label "Estimated Monthly Savings"
+        format "printf(%s (.currencyCode}}))"
```

Comment:
Hmm not sure about this -- maybe just keep it simple and make currency code a separate field.